src/lifetimes/exercise.md

```diff
@@ -35,13 +36,21 @@ implemented as a "tag" followed by the value. The tag contains a field number
 how the payload should be determined from the byte stream.
 
 Integers, including the tag, are represented with a variable-length encoding
-called VARINT. Luckily, `parse_varint` is defined for you below. The given code
-also defines callbacks to handle `Person` and `PhoneNumber` fields, and to parse
-a message into a series of calls to those callbacks.
+called VARINT. Luckily, `parse_varint` is defined for you below. The encoding of
+I32 define precisely 4 bytes in little-endian order for types `fixed32`,
```

Comment:
```suggestion
`I32` define precisely 4 bytes in little-endian order for types `fixed32`,
```

Reply:
Feel free to add it to `VARINT` above as well, though I fear this will require calling `dprint fmt` to fix the formatting :smile: